templates/zerver/integrations/github-actions.md

```diff
@@ -1,8 +1,27 @@
-It's easy to send Zulip messages from GitHub Actions! This is useful:
+# Zulip GitHub Actions integration
 
-  * If you want to report the success or failure of an Action to Zulip.
-  * If you want to send output from your actions to Zulip.
+It's easy to send Zulip messages from GitHub Actions!
 
-Instructions are available [here](https://github.com/zulip/github-actions-zulip#readme).
+{start_tabs}
 
-See also the [GitHub integration](/integrations/doc/github).
+1. {!create-a-generic-bot.md!}
+   Note down the bot's email address and API key.
+
+1. Integrate the `zulip/github-actions-zulip/send-message@v1` action into
+   your GitHub Actions workflow file, to send Zulip notifications
+   whenever your workflow runs.
+
+1. Set the input values as specified in the [README][README] of the Zulip
+   GitHub Actions repository.
+
+{end_tabs}
+
+### Related documentation
+
+* [Zulip GitHub Actions repository](https://github.com/zulip/github-actions-zulip)
+
+* [Configuring the Action][README]
+
```

Comment:
Does GitHub always capitalize Action? Our style is generally not to, but we can for consistency with them.

Reply:
Also, the link name should be more explicit, like "Configuring the Zulip Send Message Action".